modules/test/playwright/tests/journal-web/journalArticle.spec.ts

```diff
@@ -1538,3 +1568,18 @@ scheduleTest(
 		);
 	}
 );
+
+async function createAndPublishBasicArticleWithURL(
```

Comment:
This function shouldn't be on the test file. I will move it probably to `JournalEditArticlePage.ts` (may require some changes) to a function called something like `createBasicArticleWithFriendlyURL`. This way our pages works like poshi macros, all the "helper" functions are there.